enterprise-cfengine-guide/federated-reporting.markdown

```diff
@@ -571,55 +571,113 @@ you wish to disable and change the top-level `target_state` property value to `o
+     },
+     "statistics": []
+   }
+   ```
 
-Determine the id from the "id" property value and delete the remote hub. In this case
+* Determine the id from the "id" property value and delete the remote hub with the API. In this case
 we use the number "1".
 
-```console
-$ curl -k -s -X DELETE -u admin:$PASSWORD https://$SUPERHUB/api/fr/remote-hub/1
-```
+   ```console
+   $ REMOTE_HUB_ID=1
+   $ curl -k -s -X DELETE -u admin:$PASSWORD https://$SUPERHUB/api/fr/remote-hub/$REMOTE_HUB_ID
+   ```
+
+* Remove the feeder from `/opt/cfengine/federation/cfapache/federation-config.json`. Replace "id-1" below with the appropriate id from the previous steps.
+
+   ```console
+   contents=$(jq 'del(.remote_hubs ."id-1")' /opt/cfengine/federation/cfapache/federation-config.json) && echo "${contents}" > /opt/cfengine/federation/cfapache/federation-config.json
+   ```
+   
+* Remove items associated with this feeder in the `cfdb` database.
+
+    Determine the cfdb-specific `hub_id`.
+
+   ```console
+   # /var/cfengine/bin/psql cfdb -c "select * from __hubs"
+   ```
+
+   Typical output would be like the following.
+
+   ```console
+   hub_id |                               hostkey                                | last_import_ts                                                               
+   --------+----------------------------------------------------------------------+----------------                                                              
+      0 | SHA=50d370f41c81b3e119506befecc5deaa63c0f1d9039f674c68f9253a07f7ad84 |                                                                              
+      1 | SHA=bfd6f580f9d19cb190139452f068f38f843bf9227ca3515f7adfecfa39f68728 |                                      
+   (2 rows) 
+   ```
+
+   `hub_id` of `0` is the superhub. The others are the feeders.
+   In this case, it happens that the `hub_id` is also "1" so we will use that in the following queries.
+
+* Execute the following commands to remove the namespace for that feeder as well as the entry in the `__hubs` table.
+
+   ```console
+   # /var/cfengine/bin/psql cfdb -c 'drop schema "hub_1" cascade;'
+   # /var/cfengine/bin/psql cfdb -c "delete from __hubs where hub_id = 1"
+   ```
+
+* On the feeder, replace `/opt/cfengine/federation/cfapache/federation-config.json` with the following content.
+  
+  If you wish to re-add this feeder to a superhub, change "target_state" from "off" to "on".
+  Remember to trigger or wait for an agent run for the change from off to on to take effect.
+
+   ```json
+   {
+       "hostname": null,
+       "role": "feeder",
+       "target_state": "off",
+       "remote_hubs": { }
+   }
+   ```
+   
+
+* On 3.15.x and greater feeders, also run the following commands to truncate two tables:
+
+   ```console
+   # /var/cfengine/bin/psql cfsettings -c 'TRUNCATE remote_hubs'
```

Comment:
host hint in console (you do explictly say feeders though, so maybe un-necessary noise.